docs/gr/manuals/application-lifecycle.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Εγχειρίδιο κύκλου ζωής εφαρμογής Defold
+σύντομη: Αυτό το εγχειρίδιο περιγράφει τον κύκλο ζωής των παιχνιδιών και εφαρμογών Defold.
```

Comment:
Do not translate "brief:"!

Reply:
by accident i done this too , the others i fix